linter/linter.go

```diff
@@ -1,24 +1,24 @@
 package linter
 
 import (
-	"errors"
 	"fmt"
 	"golang.org/x/crypto/ocsp"
-	"time"
 )
 
-const (
-	ProducedAtLimit = "96h" // 4 days
-	ThisUpdateLimit = "96h" // 4 days
-)
+var StatusIntMap = map[int]string {
+	ocsp.Good: "good",
+	ocsp.Revoked: "revoked",
+	ocsp.Unknown: "unknown",
+	// ocsp.SeverFailed is never used:
```

Comment:
Could you provide more explanation for why ServerFailed is never used?

Reply:
Will add https://godoc.org/golang.org/x/crypto/ocsp#pkg-constants in comment